working-release-notes/template.md

```diff
@@ -0,0 +1,11 @@
+# Feature Name (remember to update the file name to match)
+
+authors = ["@FerrisTheCrab", "@BirdObsessed"]
+contributors = ["@HelpfulHelper"]
+prs = [14791, 15458, 15269]
+
+Add release note content here. Answer:
+
+- what has been changed or added?
+- why is this a big deal for users?
+- how can they use it?
```

Comment:
Zola and many existing markdown editors use a yaml metadata delimited by `---` or `+++`. Do we want to do that as well?

```suggestion
---
title: Feature Name (remember to update the file name to match)
authors = ["@FerrisTheCrab", "@BirdObsessed"]
contributors = ["@HelpfulHelper"]
prs = [14791, 15458, 15269]
---

Add release note content here. Answer:

- what has been changed or added?
- why is this a big deal for users?
- how can they use it?
```

Reply:
This makes my local mdlint angry; I don't think it's worth it.